bin/testObservability/helper/helper.js

```diff
@@ -181,178 +183,6 @@ exports.findGitConfig = (filePath) => {
   }
 }
 
-const getGitMetaData = () => {
```

Comment:
[Not a blocker] These methods have been moved to the common `helpers/helper.js` method correct?

Reply:
Yes